_layouts/docs.html

```diff
@@ -289,8 +289,19 @@
 								{% assign my_min = page.toc_min | default: site.toc_min | default: 2 %}
 								{% assign my_max = page.toc_max | default: site.toc_max | default: 3 %}
 								{% assign my_name = page.url | default: "unnamed" %}
-								<div id="side-toc"><div id="side-toc-title">On this page:</div><div id="side-toc-contents">{% include toc_pure_liquid.html html=content sanitize=true class="inline_toc" id="my_toc" toc_min=my_min toc_max=my_max page_name=my_name %}</div></div>
+								{% if my_min < 2 %}
+									{% comment %}
+										the 'content' variable doesn't include the <h1> by default
+										so if we request the h1 heading to be in the in-page TOC,
+										append it to the content before we pass it to the TOC generator
+									{% endcomment %}
+									{% assign my_content = "<h1>" | append: page.title | append: "</h1>" | append: content %}
```

Comment:
Ahhh I _think_ if an ID attr is added to this `<h1>` (i.e. make it `<h1 id="page-title">`) then the change to the `_includes/toc_pure_liquid.html` wouldn't be necessary. Just as a quick note 👍